test/cctest/node_test_fixture.h

```diff
@@ -41,8 +41,8 @@ struct Argv {
     snprintf(argv_[0], prog_len, "%s", prog);
     snprintf(argv_[0] + prog_len, arg1_len, "%s", arg1);
     snprintf(argv_[0] + prog_len + arg1_len, arg2_len, "%s", arg2);
-    argv_[1] = argv_[0] + prog_len + 1;
-    argv_[2] = argv_[0] + prog_len + arg1_len + 1;
+    argv_[1] = argv_[0] + prog_len;
+    argv_[2] = argv_[0] + prog_len + arg1_len;
```

Comment:
Can you remove this change from this PR as it's being taken care of in https://github.com/nodejs/node/pull/12110?